conda-forge.yml

```diff
@@ -4,3 +4,5 @@ github:
 conda_forge_output_validation: true
 conda_build:
   pkg_format: '2'
+azure:
+  store_build_artifacts: true
```

Comment:
Would suggest we drop this from `main` since this was primarily for debugging in the PR. Also would need a re-render

Reply:
Yerp, was going to circle back around on it tomorrow, but this merge resulted in:

```
zstd.ZstdError: zstd compress error: Allocation error : not enough memory
```

Reply:
Sounds good

Yep known issue ( https://github.com/conda/conda-package-handling/issues/167 )